src/extension.ts

```diff
@@ -79,13 +79,20 @@ class VerySmartSelect {
         this.windowSelectionListener.dispose();
     }
 
-    private adjustSelection(range: Range) {
+    private updateSelections(selections: Selection[]) {
         const editor = window.activeTextEditor;
-        if (editor === undefined) {
-            return;
+        if (editor && selections.length > 0) {
```

Comment:
Check for `selections.length > 0` because setting `editor.selections = []` causes an "Invalid arguments" error